ui/static/app/record-request/record-request-detail.component.ts

```diff
@@ -12,14 +12,18 @@ import { RecordRequest } from "./record-request";
 @Component({
   templateUrl: "/static/app/templates/record-request-detail.component.html",
   providers: [DomainService, RecordService, RecordRequestService],
-  styles: ["td span { cursor:pointer; }"]
+  styles: [`
+    td span { cursor:pointer; }
+    :host >>> .new { color: green; }
+    :host >>> .old { color: red; }
```

Comment:
Adds CSS for the Host (parent component)
It uses this style to work in InnerHTML